conf/openshift/openshift.json

```diff
@@ -105,6 +105,7 @@
         "selector": {
           "name": "iqss-dataverse-solr"
         },
+        "clusterIP": "None",
```

Comment:
Why?

Reply:
This is an unintentional change that came from a commit I cherry-picked. Removed.